app/components/enterprise_edition/banner_component.sass

```diff
@@ -51,14 +51,9 @@
     grid-template-columns: 1fr 1fr
 
     &--content
+      align-self: center
       padding: var(--base-size-16)
```

Comment:
Just out of curiosity, instead of `padding: var(--base-size-16)`, can we use system argument `p: 4` on the element?

Reply:
We could do with something like `p: medium? ? 4 : 1`. I'm not sure what we prefer internally. Pinging @HDinger for later so we can revisit